DevelopGitHub/GitHubExtension/GitHubExtension.Activity.Internal.WebApi/Controllers/InternalActivityController.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Web.Http;
+using GitHubExtension.Activity.DAL;
+using GitHubExtension.Activity.Internal.WebApi.Extensions;
+using GitHubExtension.Activity.Internal.WebApi.Mappers;
+using GitHubExtension.Activity.Internal.WebApi.Models;
+using GitHubExtension.Activity.Internal.WebApi.Queries;
+
+namespace GitHubExtension.Activity.Internal.WebApi.Controllers
+{
+    public class InternalActivityController : ApiController
+    {
+        private readonly IContextActivityQuery _contextActivityQuery;
+
+        public InternalActivityController(IContextActivityQuery contextActivityQuery)
+        {
+            _contextActivityQuery = contextActivityQuery;
+        }
+
+        [HttpGet]
+        [Route(ActivityRouteConstants.CurrentRepositoryActivityRoute)]
+        public IHttpActionResult GetCurrentRepositoryUserActivities()
+        {
+            var identity = User.Identity as ClaimsIdentity;
+            Claim currProjectClaim = identity.FindFirst("CurrentProjectId");
```

Comment:
public static class IdentityExtensions
    {
        private const string CurrentProjectId = "CurrentProjectId";

        public static Claim GetCurrentProjectClaim(this IPrincipal user)
        {
            var identity = user.Identity as ClaimsIdentity;
            Claim currProjectClaim = identity.FindFirst(CurrentProjectId);

            return currProjectClaim;
        }
    }